hypertrace-view-generator/hypertrace-view-generator/src/test/java/org/hypertrace/viewgenerator/generators/LogEventViewGeneratorTest.java

```diff
@@ -0,0 +1,99 @@
+package org.hypertrace.viewgenerator.generators;
+
+import com.google.gson.Gson;
+import java.nio.ByteBuffer;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.hypertrace.core.datamodel.Attributes;
+import org.hypertrace.core.datamodel.LogEvent;
+import org.hypertrace.core.datamodel.LogEvents;
+import org.hypertrace.core.datamodel.shared.trace.AttributeValueCreator;
+import org.hypertrace.viewgenerator.api.LogEventView;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class LogEventViewGeneratorTest {
+
+  @Test
+  public void testProcess_emptyLogEvents() {
+    LogEvents logEvents =
+        LogEvents.newBuilder()
+            .setLogEvents(Collections.singletonList(LogEvent.newBuilder().build()))
+            .build();
+
+    List<LogEventView> list = new LogEventViewGenerator().process(logEvents);
+    // empty record is generated
+    Assertions.assertFalse(list.isEmpty());
```

Comment:
I think we are trying to test with all default values of LogEvent.  And for similar properties.
```
Assertions.assertTrue(list.get(0).getTimestampNanos() == 0);
```